ci/deployment.yaml.in

```diff
@@ -55,10 +58,9 @@ metadata:
     app: hail-ci
 spec:
   ports:
-    - port: 80
+    - name: http
+      port: 80
       protocol: TCP
       targetPort: 5000
   selector:
     app: hail-ci
-  loadBalancerIP: 104.198.57.66 # hail-ci-0-1
```

Comment:
I don't follow how DNS is gonna work without static IPs?

Reply:
All DNS entries point the nginx reverse proxy now.  This isn't actually being used anymore.